imap_processing/tests/lo/test_lo_science.py

```diff
@@ -115,6 +116,39 @@ def segmented_pkts_fake_data():
     return dataset
 
 
+@pytest.fixture()
+def fake_spin_data():
+    dataset = xr.Dataset(
+        data_vars=dict(
+            num_completed=(["epoch"], np.array([0, 0])),
+            acq_start_sec=(["epoch"], np.array([0, 0])),
```

Comment:
Should this be aligned with epoch? i.e. I thinik that epoch = acq_start_sec + acq_start_subsec

Reply:
I don't think I understand. Do you mean epoch should be recalculated based on the acq_start in the packet?

Reply:
That is what other instruments are doing. If you want to stamp your data to a time, the acquisition start is usually what you want to use rather than the MET. I meant should these be `np.array([0, 1])` to align with the packet incrementing too.

Reply:
Ok sounds good, I didn't think those adjustments were being made until L1B, but that makes sense to do it here since all the data is available in the packet. For example the direct events use SHCOARSE for the Epoch for L1A, but in L1B I'm calculating a new epoch from the spin start time + de time (which is relative to the spin start time).